crates/driver/src/infra/liquidity/mod.rs

```diff
@@ -10,7 +10,30 @@
 //!    service, at which point being it will truly fit in the [`crate::infra`]
 //!    module.
 
+use {crate::domain::eth, std::cmp::Ordering};
+
 pub mod config;
 pub mod fetcher;
 
 pub use self::{config::Config, fetcher::Fetcher};
+
+/// An ordered token pair.
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub struct TokenPair(eth::TokenAddress, eth::TokenAddress);
```

Comment:
Following up from the previous comment - I'd personally probably move this into `domain` and then expose the `Fetcher` interface to work with this type.

Reply:
Love it! I changed `boundary::liquidity::Fetcher` to work on this type, which should have been a hint that I should have done the same thing for `infra::liquidity::Fetcher`. This way, I can move the decision of "what token pairs matter" to the domain, where it makes more sense anyway (otherwise, my `infra::liquidity::Fetcher` is deciding what matters and what doesn't for my `domain::Auction`).

Reply:
982c0efe6a358148cb29cf396b4957090df0a74e